src/duplicacy_snapshotmanager.go

```diff
@@ -1607,6 +1605,15 @@ func (manager *SnapshotManager) PruneSnapshots(selfID string, snapshotID string,
 
 	defer func() {
 		if logFile != nil {
+
```

Comment:
I'm not sure if it is a good idea to call Stat() before Close().  The file may have not been actually written to the disk because the Close() call, you're likely to get a NotExist error.

Reply:
Then should i call Stat after the close, even if close gives error?